torch/_dynamo/variables/torch.py

```diff
@@ -1320,6 +1352,151 @@ def speculate_branch(branch):
             )
             r = body_r.as_proxy().node.meta["example_value"]
             example_value = r
+        elif self.value is torch._functorch.eager_transforms.grad_impl:
+            # TODO: Support `fn` with kwargs.
```

Comment:
What happens when `fn` has kwargs today? Is it a graph break?

Are you planning on supporting kwargs in this PR or a separate one? Either is fine; if we don't support it in this PR then it should be at least a graph break

Reply:
Adding `graph break` for now and a related test.